src/Orleans/Configuration/ClientConfiguration.cs

```diff
@@ -175,7 +175,11 @@ public ClientConfiguration()
             NetInterface = null;
             Port = 0;
             DNSHostName = Dns.GetHostName();
+#if NETSTANDARD
+            DeploymentId = Environment.MachineName;
+#else
```

Comment:
I can do that. Nevertheless in theory there could be a side effect: if several machines didn't override the default, but since they are being run by the same service account, then all of their deployment IDs align. Not sure how important would be not to break that scenario though.
Anybody else have other opinions? or maybe a different default (or even constant) value?